src/main/java/io/servertap/PluginEntrypoint.java

```diff
@@ -45,7 +46,15 @@ public void onEnable() {
                 config.defaultContentType = "application/json";
                 config.registerPlugin(new OpenApiPlugin(getOpenApiOptions()));
                 config.showJavalinBanner = false;
+                config.accessManager((handler, ctx, permittedRoles) -> {
+                if (!bukkitConfig.getBoolean("useKeyAuth") || bukkitConfig.getString("key").equals(ctx.formParam("key"))) {
```

Comment:
Instead of `ctx.formParam("key")` we should use `ctx.header("key")` that way we don't need to pollute the form and more align with REST semantics.